yt_dlp/extractor/mlb.py

```diff
@@ -263,3 +265,68 @@ def _download_video_data(self, display_id):
   }
 }''' % display_id,
             })['data']['mediaPlayback'][0]
+
+
+class MLBTVIE(InfoExtractor):
+    _VALID_URL = r'https?://(?:www\.)?mlb\.com/tv/g(?P<id>\d{6})/?.*'
```

Comment:
```suggestion
    _VALID_URL = r'https?://(?:www\.)?mlb\.com/tv/g(?P<id>\d{6})'
```

Reply:
Neat, didn't know we don't need to match the whole UR: